indico/modules/events/registration/fields/simple.py

```diff
@@ -69,6 +69,22 @@ class TextAreaField(RegistrationFormFieldBase):
     wtf_field_class = wtforms.StringField
 
 
+def _get_places_used(field):
+    places_used = {}
+    for registration in field.form_item.registration_form.registrations:
+        if field.form_item.id not in registration.data_by_field:
+            continue
+        data = registration.data_by_field[field.form_item.id].data
+        if not data:
+            continue
+        for k, v in data.iteritems():
```

Comment:
Why not use a collections.Counter? Then you can always do `places_used[k] += v` without checking if it exists or not